### PR TITLE
feat: LightClapper node

### PR DESCRIPTION
**Pull request by marianarafaelwhite**
This change introduces the LightClapper node.

At this point, the LightClapper SW can correctly
interface with HW, send data to a ThingSpeak
channel, and can be read back into a DB.

### CODE REVIEW

File: lightclapper/lightclapperclient.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        for f in feeds:
+            date_data = f.get('created_at', '')
+            date_list = re.split('T|Z', date_data)
+
+            if len(date_list) < 2:
+                return None
+
+            data['date'] = date_list[0]
+            data['time'] = date_list[1]
+            data['lightStatus'] = f['field1']
```
Comment (marianarafaelwhite) [inline lightclapper/lightclapperclient.py]: Should update this later to be f.get('field1', None). Then do a quick check that the value is valid or exists (i.e. not None)

File: lightclapper/lightclapper.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            self.__led.invert_status()
+            toggle = True
+
+        return toggle
+
+    def __write_status_to_channel(self):
+        """
+        Write status of light clapper to channel
+        """
+        status = 1 if self.__led.get_status() else 0
```
Comment (marianarafaelwhite) [inline lightclapper/lightclapper.py]: Magic numbers; create constants to represent 1 and 0

File: lightclapper/lightclapperclient.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                'lightStatus': ''}
+
+        for f in feeds:
+            date_data = f.get('created_at', '')
+            date_list = re.split('T|Z', date_data)
+
+            if len(date_list) < 2:
+                return None
+
+            data['date'] = date_list[0]
```
Comment (marianarafaelwhite) [inline lightclapper/lightclapperclient.py]: Magic numbers here. Either add a comment to explain this better, or constants to show where 0 and 1 come from

File: lightclapper/sqliteDB.py [hunk 4]
```diff
@@ -0,0 +1,248 @@
+"""
+db.py
```
Comment (marianarafaelwhite) [inline lightclapper/sqliteDB.py]: Update file name in comments.

File: lightclapper/constants.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+L2_M_5C1_READ_KEY = 'YHMOLLMTKT2ENCK9'
+L2_M_5C1_FEED = '1150656'
+
+# ThingSpeak Channel for LightClapper testing
+L2_M_5C2_WRITE_KEY = 'L8I9PFNVXGV9K5DX'
+L2_M_5C2_READ_KEY = '1FN82DN1D6QAOIR2'
+L2_M_5C2_FEED = '1208490'
+
+# URL Syntax for LightClapper
+READ_URL = 'https://api.thingspeak.com/channels/' + \
```
Comment (marianarafaelwhite) [inline lightclapper/constants.py:23]: You can maybe experiment with updating this URL to be in the form of api.thingspeak.com/channels/CHANNEL+FEED/feeds.json?api_key=READ_KEY

**Review by marianarafaelwhite — COMMENTED** [review]
Minor comments; could be changed later

**Review by Hasan-Baig — APPROVED** [review]
Looks good to commit: 
- constants.py might move to a database or keep as is

**Comment by marianarafaelwhite** [comment]
Updated with more commits for review before merging with main branch.

**Pull request merged**